arangosh/Shell/ClientFeature.h

```diff
@@ -73,10 +77,9 @@ class ClientFeature final : public HttpEndpointProvider {
   uint64_t sslProtocol() const { return _sslProtocol; }
   bool forceJson() const { return _forceJson; }
   void setForceJson(bool value) { _forceJson = value; }
-  
-  std::unique_ptr<httpclient::GeneralClientConnection> createConnection();
+
   std::unique_ptr<httpclient::GeneralClientConnection> createConnection(std::string const& definition);
-  std::unique_ptr<httpclient::SimpleHttpClient> createHttpClient() const;
+  std::unique_ptr<httpclient::SimpleHttpClient> createHttpClient(int threadNumber = 0) const;
```

Comment:
In all other places the thread numbers are unsigned. Here they are signed. For consistency it would be better to use an unsigned type throughout.
```suggestion
  std::unique_ptr<httpclient::SimpleHttpClient> createHttpClient(int threadNumber = 0) const;
```